boards/shields/nrf54l15dk_arduino_adapter/nrf54l15dk_arduino_adapter.overlay

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/*
+ * Arduino header adapter shield for nrf54l15dk
+ * This shield maps nrf54l15dk pins to Arduino header format
+ * allowing standard Arduino shields to work on nrf54l15dk
+ */
+
+/ {
+    /* Arduino header connector definition for nrf54l15dk */
+    arduino_header: connector {
+        compatible = "arduino-header-r3";
+        #gpio-cells = <2>;
+        gpio-map-mask = <0xffffffff 0xffffffc0>;
+        gpio-map-pass-thru = <0 0x3f>;
+        gpio-map = <0 0 &gpio2 8 0>,   /* A0 - Reset pin */
+                   <1 0 &gpio2 6 0>,   /* A1 - Busy pin */
+                   <2 0 &gpio1 11 0>,  /* A2 - Event pin */
+                   <3 0 &gpio0 0 0>,   /* A3 - CS pin */
+                   <4 0 &gpio0 4 0>,   /* A4 - Extra GPIO */
+                   <5 0 &gpio0 5 0>,   /* A5 - Extra GPIO */
+                   <6 0 &gpio0 1 0>,   /* D6 - MOSI */
+                   <7 0 &gpio0 2 0>,   /* D7 - MISO */
+                   <8 0 &gpio0 3 0>,   /* D8 - SCK */
+                   <9 0 &gpio2 6 0>,   /* D9 - Busy pin */
+                   <10 0 &gpio0 0 0>,  /* D10 - CS pin */
+                   <11 0 &gpio1 11 0>, /* D11 - Event pin */
+                   <12 0 &gpio0 2 0>,  /* D12 - MISO */
+                   <13 0 &gpio0 0 0>,  /* D13 - CS pin */
+                   <14 0 &gpio0 6 0>,  /* D14 - Extra GPIO */
+                   <15 0 &gpio0 7 0>;  /* D15 - Extra GPIO */
```

Comment:
It seem not be compatible with Arduino pinout https://docs.arduino.cc/retired/boards/arduino-uno-rev3-with-long-pins/#pinout-diagram